cmd/influxd/run/server_helpers_test.go

```diff
@@ -183,6 +183,7 @@ func NewConfig() *run.Config {
 
 	c.Data.Dir = MustTempFile()
 	c.Data.WALDir = MustTempFile()
+	c.Data.WALEnableLogging = false
```

Comment:
Can you change it to `WALLoggingEnabled` to be consistent with the other `Enabled` fields?